libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -349,6 +352,17 @@ export type TigerSpecificFunctions = {
     createCSPDirective?: (requestData: ICSPDirective) => Promise<ICSPDirective>;
     updateCSPDirective?: (directiveId: string, requestData: ICSPDirective) => Promise<ICSPDirective>;
     deleteCSPDirective?: (directiveId: string) => Promise<void>;
+    getWorkspaceCustomAppSettings?: (
+        workspaceId: string,
+        applicationName?: string,
+    ) => Promise<JsonApiCustomApplicationSettingOutWithLinks[]>;
```

Comment:
We should not use `api-client-tiger` types on the interfaces, so they don't leak to our applications. (they are often changed). Better would be to create similar interface right here (less nested, then original JSON schema types, and properly documented). You can take some inspiration from `sdk-backend-spi`.